docs/src/dev_notes/standalone_ttir_run.md

```diff
@@ -0,0 +1,58 @@
+# How to run standalone MLIR, based on generated Forge-FE MLIR graphs
+
+
+1. Change Directory to tt-mlir repo in tt-forge-fe third parties
+    ```bash
+    $ cd tt-forge-fe/third_party/tt-mlir
+    ```
+
+2. Build TTRT (once) - (Inside tt-mlir repo)
+    ```bash
+    $ pip install patchelf
+    $ cmake --build build -- ttrt
+    ```
+
+3. Save system descriptor artifacts file
```

Comment:
It might make more sense to point to the [ttrt docs](https://docs.tenstorrent.com/tt-mlir/ttrt.html#generate-a-flatbuffer-file-from-compiler) for relevant info...

We would just have the instructions of how to create the initial ttir mlir file, and then for the rest point to the ttrt docs...